xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -576,6 +576,7 @@ void CVideoPlayerAudio::Process()
         msg.player = VideoPlayer_AUDIO;
         msg.cachetotal = cachetotal;
         msg.cachetime = cachetime;
+        msg.next_pts = m_messageQueue.GetTimeBack();
         msg.timestamp = m_audioClock;
```

Comment:
Any idea how to debug it most efficient?

Situation is:
- VideoPlayerVideo receives Change Message
- VideoPlayerVideo gets data from decoder and calls OutputImage wich reconfigures Renderer.
The Image is really the first one in the new resolution, everything fine until this point.

- Application.cpp calls g_graphicsContext.Flip(dirtyRegions);
Video Area is black.

There are so many places where I could look - any idea to speed this up?
Thanks!!!!!! :-)